scripts/superdesk-search/react.js

```diff
@@ -1317,6 +1361,9 @@
                                 var itemId = search.generateTrackByIdentifier(item);
                                 var oldItem = itemsById[itemId] || null;
                                 if (!oldItem || !isSameVersion(oldItem, item) || !isArchiveItemSameVersion(oldItem, item)) {
+                                    if (oldItem && oldItem.selected != null) {
+                                        item.selected = oldItem.selected;
+                                    }
                                     itemsById[itemId] = item;
```

Comment:
might be better to do `itemsById[itemId] = angular.extend({}, oldItem, item);` so we don't have to modify this in case there is another flag added